src/PageContainer.js

```diff
@@ -13,42 +15,51 @@ class PageContainer extends Component {
         super(props);
 
         this.state = {
-            content: props.content
+            content: props.content,
+            account: props.account
```

Comment:
shouldn't save `props.account` as a `state` here, it won't get updated, so we should always just pass it down as props from props (plus you're not using it either)